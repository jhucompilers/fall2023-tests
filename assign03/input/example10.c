int main(void) {
  int a[3];
  int *p;

  p = &a[0] + 2; // pointer arithmetic should be allowed
  *p = 42; // assigns to a[2]

  return 0;
}
