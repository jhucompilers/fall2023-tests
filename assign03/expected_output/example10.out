0|main|function|function () returning int
2|a|variable|array of 3 x int
2|p|variable|pointer to int
